src/Config/Registrar.php

```diff
@@ -18,9 +18,16 @@
 class Registrar
 {
     /**
+     * --------------------------------------------------------------------------
+     * Shield OAuth Decorator
+     * --------------------------------------------------------------------------
      * Register the ShieldOAuth decorator.
+     *
+     * @see https://codeigniter.com/user_guide/outgoing/view_decorators.html
+     *
+     * @return array<string, array<int, string>>
```

Comment:
PHPStan has lists now too! This is slightly more specific (both are fine though):
```suggestion
     * @return array<string, list<string>>
```

Reply:
Thanks 👍 
Anybody see [List](https://phpstan.org/writing-php-code/phpdoc-types#lists) for more info.